package.json

```diff
@@ -60,6 +61,7 @@
     "google-auth-library": "^5.5.0",
     "google-gax": "^1.7.5",
     "is-stream-ended": "^0.1.4",
+    "js-green-licenses": "^1.1.0",
```

Comment:
Will this still work as a devDependency?

Reply:
Err... I don't know why that even ended up as a regular dependency. I must've flubbed the command line parameter. Good catch though.

Per Justin's comments I'm probably just removing that anyhow.